CesiumGltf/src/ExtensibleObject.cpp

```diff
@@ -0,0 +1,22 @@
+#include "CesiumGltf/ExtensibleObject.h"
+
+using namespace CesiumGltf;
+using namespace CesiumUtility;
+
+JsonValue* ExtensibleObject::getGenericExtension(
+    const std::string& extensionName) noexcept {
+  return const_cast<JsonValue*>(
+      const_cast<const ExtensibleObject*>(this)->getGenericExtension(
```

Comment:
I think the claim on StackOverflow that `const_cast` can't be used to add constness is incorrect, based on my reading of https://en.cppreference.com/w/cpp/language/const_cast.

> lvalue of any type T may be converted to a lvalue or rvalue reference to the same type T, more or less cv-qualified.

But an answer further down mentions `std::as_const` which seems like the most readable and correct thing to use (since C++17).